main.go

```diff
@@ -76,36 +99,18 @@ func main() {
 		metav1.NamespaceNone,
 	}
 
-	// Get a config to talk to the apiserver
-	cfg, err := config.GetConfig()
-	if err != nil {
-		klog.Exit(err.Error())
-	}
-
-	// Create a new Cmd to provide shared dependencies and start components
-	mgr, err := manager.New(cfg, manager.Options{
-		NewCache:                cache.MultiNamespacedCacheBuilder(namespaces),
-		LeaderElection:          true,
-		LeaderElectionID:        "performance-addon-operators",
-		LeaderElectionNamespace: namespace,
-		Namespace:               namespace,
-		MetricsBindAddress:      fmt.Sprintf("%s:%d", metricsHost, metricsPort),
+	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+		NewCache:           cache.MultiNamespacedCacheBuilder(namespaces),
+		Scheme:             scheme,
+		MetricsBindAddress: metricsAddr,
+		Port:               webhookPort,
+		LeaderElection:     enableLeaderElection,
+		LeaderElectionID:   "performance-addon-operators",
```

Comment:
should be a constant. Also, we should use the singluar form

Reply:
Auto generated , will change

Reply:
that reminds me that we should always use "performance-addon-operator" (singular) where feasible. Actually not sure what we used before. If we used the plural form previously as leaderElectionID:
1. let's keep it that way
2. let's add a comment about why we keep the plural form

Reply:
Took care and re-pushed